### DISCUSSION TITLE
Should we have a 'good second issue' label

### DISCUSSION THREAD

**Discussion by damithc** [TEAMMATES/teammates#11163] [Q&A]
What do you guys think of having a `good second issue` label, for issues that are not high priority) and the difficulty is estimated to be just a bit beyond first timers?

**Comment by moziliar** [comment]
I was wondering about that too, after guiding some to completing their first issues.

I suppose such issue would be good; meanwhile, the mini induction project (asking new dev to add a data field or a small feature tweak) is still worth considering in parallel.

**Comment by damithc** [comment]
Thanks @moziliar 

I've added the label. Let's see how it goes. We can get rid of it later if it doesn't prove useful. Feel free to apply it to suitable issues.
